src/libs/actions/ReimbursementAccount/resetFreePlanBankAccount.js

```diff
@@ -24,6 +24,17 @@ function resetFreePlanBankAccount(bankAccountID) {
         },
         {
             optimisticData: [
+                {
+                    onyxMethod: CONST.ONYX.METHOD.MERGE,
+                    key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                    value: {
+                        shouldShowResetModal: false,
```

Comment:
No, because that would open the confirmation modal again making the error less visible:

![image](https://user-images.githubusercontent.com/87341702/222861061-34c6f2cd-2b69-4045-844b-781daa8d6f85.png)

It is a worse user experience.